resource_management_mobile/src/app/pages/resource-requirement/add-resource-requirement/add-resource-requirement.component.scss

```diff
@@ -0,0 +1,55 @@
+@import "../../../../theme/theme.scss";
+
+ion-label>span {
+  color: red;
+}
+
+.skill_container {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10px;
+}
+
+.skil-del {
+  display: flex;
+  justify-content: end;
+}
+
+.skill-add {
+  display: flex;
+  justify-content: center;
+}
+
+.add-icon {
```

Comment:
These custom classes already have in global.scss file use them...

Reply:
done

Reply:
fixed